homeassistant/components/automation/__init__.py

```diff
@@ -111,6 +111,8 @@ def is_on(hass, entity_id):
     """Set up the automation."""
     component = EntityComponent(_LOGGER, DOMAIN, hass,
                                 group_name=GROUP_NAME_ALL_AUTOMATIONS)
+    device_automation = importlib.import_module('.device_automation', __name__)
```

Comment:
I think that we should have this live under its own integration. Not be part of automation.